src/alfasim_sdk/alfasim_sdk_api/detail/bootstrap_win.h

```diff
@@ -68,56 +68,74 @@ inline int alfasim_sdk_open(ALFAsimSDK_API* api)
     full_filepath[0] = '\0';
     StringCchCatW(full_filepath, MAX_PATH_SIZE, alfasim_executable_dir);
     StringCchCatW(full_filepath, MAX_PATH_SIZE, DLL_FILENAME);
+
+    // https://learn.microsoft.com/en-us/windows/win32/api/libloaderapi/nf-libloaderapi-loadlibraryw
+    // > If the function fails, the return value is NULL.
     api->handle = LoadLibraryW(full_filepath);
     free(executable_dir_from_env);
+    if (api->handle == NULL) {
+        return SDK_FAILED_TO_LOAD_DLL;
+    }
+
+    // https://learn.microsoft.com/en-us/windows/win32/api/libloaderapi/nf-libloaderapi-getprocaddress
+    // > If the function fails, the return value is NULL.
+    #define LOAD_SDK_PROC_MAP_TYPE(func_name, func_type) {\
+        api->func_name = (func_type)GetProcAddress(api->handle, #func_name);\
+        if (api->func_name == NULL) {\
+            FreeLibrary(api->handle);\
```

Comment:
Same comment here.

Reply:
"Same" (windows not linux, but the logic is the same).